### ISSUE TITLE
Malloc error when training LLDA model

### ISSUE THREAD

**Issue by FLawrence** [bab2min/tomotopy#33]
System: OSX (10.14.6)
Version: Tomotopy 0.5.2

Example code (mostly from lda example and using the same text file) :

```import sys
import tomotopy as tp
from random import seed
from random import randint

seed(1)

def llda_example(input_file, save_path):
    topics = ['technology', 'art', 'economics', 'politics', 'religion', 'sport']
    mdl = tp.LLDAModel(tw=tp.TermWeight.ONE, min_cf=3, rm_top=5, k=20)
    for n, line in enumerate(open(input_file, encoding='utf-8')):
        ch = line.strip().split()
        labels = []
        for i in range(randint(1, 6)):
            labels.append(topics[randint(0, 5)])
        mdl.add_doc(ch,labels)
    mdl.burn_in = 100
    mdl.train(0)
            
print('Running LLDA')
llda_example('enwiki-stemmed-1000.txt', 'test.lda.bin')```

Result: Kernal dies - following message in console:
python(28393,0x111d1b5c0) malloc: *** error for object 0x7fc6f98cdfe0: pointer being freed was not allocated
python(28393,0x111d1b5c0) malloc: *** set a breakpoint in malloc_error_break to debug

**Comment by bab2min** [comment]
Hi @FLawrence, thank you for reporting the bug.
Actually I couldn't reproduce the same issue, but I found some bugs in the code of initializing `LLDAModel` at version 0.5.2. I think it may cause the memory allocation problem. 
Since this fix has been included in version 0.6.2, please re-try in recent version.
If you meet similar bugs, please report it again.
Thank you!

**Comment by FLawrence** [comment]
Retried with version 7.0.0 and seems to be working. Thanks!
